integrations/salesforce.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Salesforce'
+---
+
+To connect your Salesforce account, head to **Settings** → **Salesforce integration** and follow the instructions. It will take a couple of clicks.
```

Comment:
```suggestion
To connect your Salesforce account, head to **Settings** → **Salesforce importer** and follow the instructions. It will take a couple of clicks.
```